config/state.ts

```diff
@@ -105,6 +127,9 @@ type Action = {
 } | {
 	type: 'SET_SFX_CONFETTI';
 	payload: boolean;
+} | {
+	type: 'SET_SOUND_ENABLED';
```

Comment:
Please update this action type to `SET_SFX_SOUND` to follow the naming convention already set.